### PR TITLE
Head of Personnel Resprite & New HoP Coat

### PR DESCRIPTION
**Pull request by axietheaxolotl**
## About The Pull Request

You can read the title, can't you? This should be the first part of a whole resprite of the Service department (the best department).

image

As you can very graciously see in that screenshot, the HoP, now has a brand new coat to wear. Which is a lot more interesting then just a plain armour vest, which this replaces.

Also, also, I resprited the ID Painter. Why? idk cause I could.

## Why It's Good For The Game

Stuff look better = good

## Changelog

:cl:
add: Brand new HoP coat!
imageadd: New sprites for HoP's uniform, skirt, cap, and the ID Painter.
fix: HoP starts with laceups, not brown sneakers.
/:cl:

### CODE REVIEW

**Comment by axietheaxolotl** [comment]
image
image

**Comment by axietheaxolotl** [comment]
and my fucking git is screwed up kill me now

**Comment by Sheits** [comment]
I think the shadow on the lower backs, especially the cloak is too big. The arch of the back could cast a shadow like that in the middle with super dramatic lighting but it flattens out the whole sprite in my opinion, and is also inconsistent with the side sprites. Same deal with the front facing state on the jumpsuit, though it does make sense on the jumpskirt with the breasts casting a shadow, the side sprites feel inconsistent. Overall super good work thoe

**Comment by Likteer** [comment]
Colors not vibrant enough

**Comment by VladinXXV** [comment]
I think the coat looks kinda weird but I'm diggin the suit and PDA, good stuff.

**Review by Imaginos16 — APPROVED** [review]
Disregarding that minor nitpick, the sprites are most definitely good. Nice work.

**Comment by Pepsilawn** [comment]
Alt should be changed in code to keep the chest covered if it will only fold in the sleeves.
image

**Comment by CRITAWAKETS** [comment]
Why is the back of the coat unshaded?

**Comment by MrDoomBringer** [comment]
Definitely an improvement, though the dark red accents make me think this is some kind of sec-related role. Maybe change to a brown-shade, as that to me evokes cargo/service? Just a thought.

**Comment by castawaynont** [comment]
a bit evocative of the warden's jacket, looks good

**Comment by SethLafuente** [comment]
I like this sprite it looks new other than the fact that the hair sticks out of the hat probably a separate issue unrelated to the PR.

I like the current HoP suit couldn't this be an alternative suit like Captain, HoS and RD have alt ones I always opened the HoP dress thing and find it to be a bit small compared to other heads so I see no reason why adding more outfits is a bad idea.

I would say otherwise if this was a remaster of the old suit but this is pretty much an entirely different suit

File: code/modules/clothing/suits/armor.dm [hunk 1]
```diff
@@ -371,3 +371,10 @@
 /obj/item/clothing/suit/armor/centcom_formal/Initialize(mapload)
 	. = ..()
 	AddComponent(/datum/component/toggle_icon)
+
+/obj/item/clothing/suit/armor/vest/hop
+	name = "\improper Head of Personnel's coat"
```
Comment (tralezab) [inline code/modules/clothing/suits/armor.dm:376]:
```suggestion
	name = "head of personnel's coat"
```

**Review by tralezab — CHANGES_REQUESTED** [review]

**Comment by tralezab** [comment]
i'll just commit that for you and merge once checks pass

**Pull request merged**